nova/virt/vmwareapi/vmops.py

```diff
@@ -2182,9 +2182,13 @@ def _clean_shutdown(self, instance, timeout, retry_interval):
 
             LOG.debug("Soft shutdown instance, timeout: %d",
                      timeout, instance=instance)
-            self._session._call_method(self._session.vim,
-                                       "ShutdownGuest",
-                                       vm_ref)
+            try:
+                self._session._call_method(self._session.vim,
+                                           "ShutdownGuest",
+                                           vm_ref)
+            except vexc.ToolsUnavailableException:
+                LOG.exception("Failed to _clean_shutdown the instance")
```

Comment:
I don't think this should be an exception. We handle it gracefully, so there should imho be at max a warning, I'd even use "INFO" for this.

I think we have to pass `instance=instance` here, so we actually know "the instance"?